### PR TITLE
Delete old download logs command

### LINKED ISSUE #41: Delete older download logs
DBs in PaaS world are a bit less flexible on account of we pay for what we use; therefore need to limit the amount of download logs stored.

We have 1GB database, currently using "60%" (ClearDB on Heroku is not exactly clear about how much data that is exactly...).

Maybe 6mo or 12mo is a reasonable cutoff?

### PR DESCRIPTION
**Pull request by asgrim**
Fixes #41

### CODE REVIEW

File: src/Command/GenerateStatisticsCommand.php [hunk 1]
```diff
@@ -16,9 +16,9 @@ final class GenerateStatisticsCommand extends Command
      */
     private $analyseStatistics;
 
-    public function __construct(AnalyseStatistics $analyseStatistics)
+    public function __construct(AnalyseStatistics $deleteOldDownloadLogs)
```
Comment (mimmi20) [inline src/Command/GenerateStatisticsCommand.php]: Could you revert this change?

Comment (asgrim) [reply src/Command/GenerateStatisticsCommand.php]: hah, yes must've missed that in PhpStorm's refactoring :)
